Solutions/Azure Active Directory/Analytic Rules/Brute Force Attack against GitHub Account.yaml

```diff
@@ -52,6 +52,6 @@ entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: AccountCustomEntity
-version: 1.0.0
+        columnName: UserPrincipalName
```

Comment:
Fullname is not a strong entity identifier for Account (its actually a legacy mapping) where we have a UPN we should split in into Name and UPN suffix and have both as entities - https://learn.microsoft.com/en-us/azure/sentinel/entities-reference